modules/network-observability-flowcollector-example.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/configuring-operators.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-flowcollector-example_{context}"]
+= Example of a FlowCollector resource
+
+[role="_abstract"]
+Review a comprehensive, annotated example of the `FlowCollector` custom resource demonstrating configurations for `eBPF` sampling, conversation tracking, Loki integration, and console quick filters.
```

Comment:
I see the [what, where is the why](https://docs.google.com/presentation/d/1cl5PFL0SRV7M6GHBJOZ1jNMAtbVI_85iHKe5XWMV0ek/edit?slide=id.g37974b26b7e_0_84#slide=id.g37974b26b7e_0_84)?

Reply:
Seems a bit simpler.
```suggestion
[role="_abstract"]
Review a comprehensive, annotated example of the `FlowCollector` custom resource that demonstrates configurations for `eBPF` sampling, conversation tracking, Loki integration, and console quick filters.
```